tests/integration/nupic/opf/opf_experiment_results_test.py

```diff
@@ -55,7 +56,7 @@ def testExperimentResults(self):
     examples/prediction directory in the install tree (same as predictionDir)
 
     """
-    nupicDir = os.environ['NUPIC']
+    nupicDir = os.path.join(NUPIC_ROOT, "..")
```

Comment:
which module would you specify here ? neither examples nor opf are importable....